### ISSUE TITLE
Texture resolution question

### ISSUE THREAD

**Issue by kyavichev** [cnr-isti-vclab/nexus#61]
Hi there! Would I be wrong to assume that nexus limits the size of the internal textures to 4096? I took a peak at the code, and that's the value I see in the texpyramid.h. (And generally that's the size of texture that modern mobile devices support.) If that's not the case, is there a way to specify this limit?

**Comment by ponchio** [comment]
Yes, you are right. Actually I didn't pay to much attention in how to deal with very large textures...

It was supposed to be a temporary solution, unfortunately it's taking more time to develop the new one. Anyway, you could just change the value in texpyramid.h... 

That value is NOT a limit or an indication of how large a texture can be loaded in GPU, just of the texture pyramid size.
